gtk4/src/bitset_iter.rs

```diff
@@ -0,0 +1,167 @@
+// Take a look at the license at the top of the repository in the LICENSE file.
+
+use crate::Bitset;
+use glib::translate::*;
+
+#[derive(Copy, Clone)]
+pub struct BitsetIter(ffi::GtkBitsetIter);
+
+impl BitsetIter {
+    #[doc(alias = "gtk_bitset_iter_init_at")]
+    pub fn init_at(set: &Bitset, target: u32) -> Option<(Self, u32)> {
```

Comment:
These constructor names are a bit strange

Reply:
No ideas how to make them any better :-)

Reply:
Me neither. I'm not good at coming up with good names ;)

Reply:
Let's leave it as is then :-)